examples/appdir/next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   reactStrictMode: true,
   transpilePackages: ["uploadthing", "@uploadthing/react"],
+  experimental: { serverActions: true },
```

Comment:
We may want to include a note in the docs about this?

Reply:
Docs added.